### PR TITLE
Fix description of "Enforce expiration date" parameter

### PR DESCRIPTION
**Pull request by danxuliu**
The `Enforce expiration date` parameter [enforces that the expiration date will not be set further in the future](https://github.com/nextcloud/server/blob/de7026f6e4662ff42e0a3c12021a3cee25c5d91f/lib/private/Share20/Manager.php#L494-L507), but [it does not prevent users from setting a more recent date](https://github.com/nextcloud/documentation/blob/5146a352e197ef59b8d232f3f5956b73d1a3c451/admin_manual/configuration_files/file_sharing_configuration.rst#distinguish-between-max-expiration-date-and-default-expiration-date), or from updating the expiration date again at a later point.

Wording and styling improvements are welcome, I do not like too much how I wrote it :-P

### CODE REVIEW

File: admin_manual/configuration_files/file_sharing_configuration.rst [hunk 1]
```diff
@@ -15,7 +15,12 @@ Configure your sharing policy on your Admin page in the Sharing section.
 * Check ``Set default expiration date for shares`` to set a default expiration date
   on local user and group shares.
 * Check ``Enforce expiration date`` to always enforce the configured expiration date
-  on local user and group shares. Users will not be able to adjust or remove the expiration date.
+  on local user and group shares. Users will not be able to set the expiration date further
+  in the future than the enforced expiration date, although they will be able to set a more
+  recent date. Also note that they will also be able to update the expiration date again at
```
Comment (wiswedel) [inline admin_manual/configuration_files/file_sharing_configuration.rst:20]: style annotation: avoid double "also"

File: admin_manual/configuration_files/file_sharing_configuration.rst [hunk 2]
```diff
@@ -26,7 +31,12 @@ Configure your sharing policy on your Admin page in the Sharing section.
 * Check ``Set default expiration date for link shares`` to set a default expiration date on 
   public shares.
 * Check ``Enforce expiration date`` to always enforce the configured expiration date
-  on public shares. Users will not be able to adjust or remove the expiration date.
+  on public shares. Users will not be able to set the expiration date further
+  in the future than the enforced expiration date, although they will be able to set a more
+  recent date. Also note that they will also be able to update the expiration date again at
```
Comment (wiswedel) [inline admin_manual/configuration_files/file_sharing_configuration.rst:36]: style annotation: avoid double "also"

File: admin_manual/configuration_files/file_sharing_configuration.rst (hunk 1 above)
Comment (wiswedel) [inline admin_manual/configuration_files/file_sharing_configuration.rst:20]: Allow a new paragraph here for better readability

File: admin_manual/configuration_files/file_sharing_configuration.rst (hunk 2 above)
Comment (wiswedel) [inline admin_manual/configuration_files/file_sharing_configuration.rst:36]: Allow a new paragraph here for better readability

**Review by wiswedel — APPROVED** [review]
Thanks for the clarifications, @danxuliu. I made some style annotations but the message is 👍

File: admin_manual/configuration_files/file_sharing_configuration.rst (hunk 1 above)
Comment (danxuliu) [reply admin_manual/configuration_files/file_sharing_configuration.rst:20]: That was my original idea :-) But I do not know the syntax, and thus how to add a new paragraph that is shown in the list item. Suggestions welcome :-)

Comment (wiswedel) [reply admin_manual/configuration_files/file_sharing_configuration.rst]: I pushed a new revision with the note moved into a _Note_ box.

**Comment by wiswedel** [comment]
@danxuliu Please have a look at my latest changes and feel free to merge the PR if you're satisfied.

**Comment by danxuliu** [comment]
> Please have a look at my latest changes and feel free to merge the PR if you're satisfied.

I have removed an _also_ that was still left :-)

Besides that I do not know if you still want the extra paragraph for `Also note that user will be able to...`, but if you do then a new line needs to be added between [`recent date.` and `Also note`](https://github.com/nextcloud/documentation/pull/5411/files#diff-9599eb69fba5ffb627134c9417675d858aab98182fd87d2e666e25ca70f0bbc7R22-R23) (I have now checked the syntax ;-) ). 

Other than that, :+1:

**Comment by wiswedel** [comment]
Thanks for fixing the glitch and for checking the syntax, @danxuliu.

> if you still want the extra paragraph

With the whole update moved into a note box, it's readable enough for me.

**Comment by wiswedel** [comment]
@danxuliu I have no idea how to handle that fixupbot message. Could you please take care of that and then merge?

**Comment by danxuliu** [comment]
> I have no idea how to handle that fixupbot message.

For reference what is happening is that I did not create a _normal_ commit to remove the extra `also`. Instead I created a _fixup_ commit. _fixup_ commits are just temporary commits that are used during development and which then get squashed on the commit they fix before the branch is merged. Thanks to that instead of a commit and a commit that fixes the previous one you just end with a single commit and a cleaner commit history :-)

> Could you please take care of that and then merge?

Of course, done!

**Pull request merged**